packages/features/insights/server/trpc-router.ts

```diff
@@ -71,7 +71,22 @@ const buildBaseWhereCondition = async ({
         id: true,
       },
     });
-    if (teamsFromOrg.length === 0) return { whereCondition, isEmptyResponse: true };
+
+    if (teamsFromOrg.length === 0) {
+      return {
+        whereCondition: {
+          ...whereCondition,
+          OR: [
+            ...(whereCondition.OR ?? []),
+            {
+              teamId: ctx.userOrganizationId,
+              isTeamBooking: true,
+            },
+          ],
+        },
+        isEmptyResponse: true,
```

Comment:
Not really sure if we want to keep it or ditch it as I couldn't find anywhere we were using it.

Reply:
maybe follow up, we don't use it.